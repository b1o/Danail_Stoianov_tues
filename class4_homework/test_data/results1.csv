Venelin_Yonchev_1.rb,"300.00,163.00,137.00",true
Denis_Kalfov_1.rb,"300.00,163.00,137.00",true
Kristian_Popov_1.rb,"300.00,163.00,137.00",true
Ivaylo_Markov_1.rb,"300.00,163.00,137.00",true
Alexander_Bonev_1.rb,"300.00,163.00,137.00",true
Stoyan_Stoyanov_1.rb,"1426.00,861.00,565.00",false
Hristiqn_Zarkov_1.rb,"300.00,163.00,137.00",true
Bogomil_Kolarov_1.rb,"300.00,163.00,137.00",true
Yulian_Razkazov_1.rb,"300.00,163.00,137.00",true
Ivaylo_Hristov_1.rb,"300.00,163.00,137.00",true
Georgi_Puhalev_1.rb,"300.00,163.00,137.00",true
Kaloian_Pavlov_1.rb,"300.00,163.00,137.00",true
Stanislav_Beregov_1.rb,"300.00,163.00,137.00",true
Kaloyan_Stoykov_1.rb,"300.00,163.00,137.00",true
Alexander_Iliev_1.rb,"300.00,163.00,137.00",true
Maksimiliyan_Bogdanov_1.rb,"300.00,163.00,137.00",true
Metodi_Todorinov_1.rb,"300.00,163.00,137",false
Asen_Panayotov_1.rb,"300.00,163.00,137.00",true
Mihail_Kirilov_1.rb,"300.00,163.00,137.00",true
Alexandar_Apostolov_1.rb,"300.00, 163.00, 137.00",false
Ivan_Zorov_1.rb,"300.00,163.00,137.00",true
Dimitar_Velev_1.rb,"300.00,163.00,137.00",true
Georgi_Dimitrov_1.rb,"300.00,163.00,137.00",true
Danail_Stoianov_1.rb,"300.00,163.00,137.00",true
Sedef_Hristov_1.rb,"300.00,163.00,137.00",true
Alexander_Monov_1.rb,"300.00,163.00,137.00",true
Anton_Georgiev_1.rb,"300.00,163.00,137.00",true
Stefan_Stamenov_1.rb,"300.00,163.00,137.00",true
Martin_Ivanov_1.rb,"300.00,163.00,137.00",true
Daniel_Iliev_1.rb,"",false
Luben_Temelakiev_1.rb,"300.00,163.00,137.00",true
Kiril_Kostadinov_1.rb,"300.00,163.00,137.00",true
Georgi_Ivanov_1.rb,"300.00,163.00,137.00",true
Bozhidar_Nikolov_1.rb,"98.00, 0.00, 98.00",false
Nedelcho_Delchev_1.rb,"300.00,163.00,137.00",true
Spas_Spasov_1.rb,"300.00,163.00,137.00",true
Teodor_Draganov_1.rb,"",false
Milko_Filipov_1.rb,"300.00,163.00,137.00",true
Emil_Dudev_1.rb,"300.00,163.00,137.00",true
Martin_Stoyanov_1.rb,"",false
Pelovski_Pelovski_1.rb,"0.00,0.00,0.00",false
Luboslav_Ivanov_1.rb,"300.00,163.00,137.00",true
Georgi_Iliev_1.rb,"300.00,163.00,137.00",true
Presian_Abarov_1.rb,"300.00,163.00,137.00",true
Vladimir_Gekov_1.rb,"",false
Asen_Stoilov_1.rb,"300.00,163.00,137.00",true
Nedko_Boshkilov_1.rb,"300.0,163.0,137.0",false
