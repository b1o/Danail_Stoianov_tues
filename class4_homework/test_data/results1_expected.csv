Александър Ангеловски,0,0
Александър Апостолов,0,0
Александър Бонев,0,0
Александър Илиев,0,0
Александър Монов,0,0
Антон Георгиев,0,0
Асен Панайотов,0,0
Асен Стоилов,0,0
Богомил Коларов,0,0
Божидар Димитров,0,0
Васил Момчилов,0,0
Венелин Йончев,1,0
Виктор Славов,0,0
Владимир Геков,0,0
Георги Димитров,0,0
Георги Иванов,0,0
Георги Илиев,0,0
Георги Кабаджов,0,0
Георги Пухалев,0,0
Данаил Стоянов,0,0
Даниел Илиев,0,0
Денис Калфов,1,0
Димитър Велев,0,0
Емил Дудев,0,0
Ивайло Марков,0,0
Ивайло Христов,0,0
Иван Зоров,0,0
Иво Иванов,0,0
Калоян Павлов,0,0
Калоян Стойков,0,0
Кирил Костадинов,0,1
Кристиян Попов,1,0
Любен Темелакиев,0,0
Любослав Иванов,0,0
Максимилиян Богданов,0,0
Мартин Димчев,0,0
Мартин Иванов,0,0
Мартин Кузев,0,0
Мартин Стоянов,0,0
Методи Тодоринов,0,0
Милко Филипов,0,0
Михаил Кирилов,0,0
Неделчо Делчев,0,0
Недко Бошкилов,0,0
Никола Будинов,0,0
Николай Димитров,0,0
Николай Дионисов,0,0
Петър Тошков,0,0
Пресиан Абаров,0,0
Седеф Христов,0,0
Спас Спасов,0,0
Станислав Берегов,0,0
Стефан Стаменов,0,0
Стоян Стоянов,0,0
Теодор Драганов,0,0
Християн Зарков,0,0
Цветелин Истаков,0,0
Юлиян Расказов,0,0
